tests/hikari/test_commands.py

```diff
@@ -36,7 +36,7 @@ def mock_app():
 class TestCommand:
     @pytest.fixture()
     def mock_command(self, mock_app):
-        return commands.Command(
+        return commands.SlashCommand(
```

Comment:
When it comes to testing classes with abstract methods, they should use `hikari_test_helpers.mock_class_namespace` to be able to instanciate `PartialCommand`.

Additionally (github doesnt let me comment on deleted lines), the class should be named `TestPartialCommand`